packages/redirect-parser/tests/helpers/main.js

```diff
@@ -1,6 +1,6 @@
 import { fileURLToPath } from 'url'
 
-import { parseAllRedirects } from '../../lib/index.js'
+import { parseAllRedirects } from '../../src/index.js'
```

Comment:
move this file to typescript as well

Reply:
This is already updated. Github showing the old version of the file.